src/main/java/net/minecraftforge/event/world/FoundChunksForSpawningEventBuilder.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016-2018.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.event.world;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import net.minecraft.entity.EnumCreatureType;
+import net.minecraft.util.math.ChunkPos;
+import net.minecraft.world.WorldServer;
+
+public class FoundChunksForSpawningEventBuilder
```

Comment:
`FoundChunksForSpawningEvent.Builder` - nest this in the event class.

Reply:
Fixed at f851630e2de34c872cb7cda0c35b1e18c2a53c1c